### ISSUE TITLE
Using mathjax locally

### ISSUE THREAD

**Issue by e4r** [davidshen84/ngx-mathjax#17]
Is it possible to somehow configure this package to load Mathjax locally?

**Comment by davidshen84** [comment]
No, I think I hardcode that part to load from MathJax's CDN. But you can configure which version of MathJax to load. BTW, I think you need 2.x version, I did not test it with the new 3.x version. Thanks!
